src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs

```diff
@@ -1272,6 +1291,38 @@ private static bool IsExpectedException(Exception e, object paramValue, string e
                 return false;
             }
         }
+                
+        private static bool IsExpectedInvalidOperationException(Exception e, string expectedBaseTypeName)
+        {
+            if ((e.GetType() == typeof(InvalidOperationException)) &&
+                    (expectedBaseTypeName == "time") &&
+                    (e.Message.Contains("The given value ")))
+            {
+                return true;
+            }
+
+            return false;
+        }
+
+        private static string AmendTheGivenMessageDateValueExeption(string message, object paramValue)
```

Comment:
Nice!

Reply:
Although I just noticed a typo in the method name:
`AmendTheGivenMessageDateValueExeption` -> `AmendTheGivenMessageDateValueException`